podoglyph/gradebook_v
# Development

#### What does this PR do?
This PR brings student course, grades, and GPA to the student dashboard. Currently exposes only the current semester's enrollments.

#### How should this be manually tested?
Login as a student and you should see on the student dashboard a list of current enrollments, grades for each course, and the overall GPA for the current semester.

#### Any background context you want to provide?

#### Questions:

  - Do Migrations Need to be ran?
* Yes. Added current:boolean to Semester table.

  - Do Environment Variables need to be set?
* No
  - Any other deploy steps?
* `db:seed` to get updated migration data.
